emacs/radian.el

```diff
@@ -1007,16 +1006,17 @@ ourselves."
 
   ;; The default settings seem a little forgetful to me. Let's try
   ;; this out.
-  (setq prescient-history-length 1000))
+  (setq prescient-history-length 1000)
 
-;; Package `selectrum-prescient' provides intelligent sorting and
-;; filtering for candidates in Selectrum menus.
-(radian-use-package selectrum-prescient
-  :demand t
-  :after selectrum
-  :config
+  ;; Use prescient.el everywhere.
+  (setq completion-styles '(prescient))
+
+  ;; Common sense.
+  (setq prescient-sort-full-matches-first t)
 
-  (selectrum-prescient-mode +1))
+  ;; If we don't do this then prescient.el sorting is not used in most
+  ;; places.
+  (setq prescient-completion-enable-sort t))
```

Comment:
After experimenting with this, I thought that it would be easier to configure the UIs with this disabled. For example, with this and Company Prescient enabled, any candidates from CAPFs get the frecency sorting applied twice. I thought about trying to detect when a CAPF applied the sorting, but it didn't feel robust. There's also the matter of `company-prescient-sort-length-enable`, which is harder to apply for CAPFs that can be run from many UIs.

I think that leaving this as `nil` and setting `vertico-sort-function` to `#'prescient-completion-sort` is better, since it only affects Vertico and still applies sorting to candidates from other completion styles (such as `basic`as mentioned above).